Epicure-Front/src/components/Common/Header/NavBar/styledNavBar.ts

```diff
@@ -44,11 +44,11 @@ export const OpenHamburgerLinksButton = styled.button`
   font-size: ${fontSizes.large};
 
   @media (max-width: 600px) {
-    display: block; /* Show the button for screens 600px or narrower */
+    display: block;
   }
 
   @media (min-width: 601px) {
-    display: none; /* Hide the button for screens wider than 600px */
+    display: none;
   }
```

Comment:
remember to use consts for media queries instead of using hard coded mesurements.